website/client/js/controllers/filtersCtrl.js

```diff
@@ -17,6 +17,7 @@ habitrpg.controller("FiltersCtrl", ['$scope', '$rootScope', 'User', 'Shared',
             User.updateTag({params:{id:tag.id}, body:{name:tag.name}});
         })
         $scope._editing = false;
+        User.sync();
```

Comment:
When tags are deleted though, it takes a sync for the tagged icon to disappear from tasks if there are none left once it is deleted. Is there a less expensive way to update these?